src/components/CookieBar/CookieBtn.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Button} from '../../index'
+
+const CookieBtn = props => {
+    return (
+        <span>
+            <Button color="link">{props.buttonName}</Button>
```

Comment:
It is correct to leverage the `Button` component, but I think you have to stick to the reference HTML.
In particular the `Button` component injects a `btn` class that it needs to be overwritten - see the `cssModules` prop in reactstrap: https://reactstrap.github.io/components/buttons/ .
Also you can leverage the `tag` prop of the `Button` component for reject and accept version as in the reference documentation.